frontend/src/index.js

```diff
@@ -55,6 +56,7 @@ ReactDOM.render(
                 <Route path={paths.healthMeasurements} component={HealthMeasurements} />
                 <Route path={paths.signUp} component={SignUp} />
                 <Route path={paths.myStory} component={MyStory} />
+                <Route path={paths.feeling} component={Feeling} />
```

Comment:
Ooh I see you've picked on the `paths` constant! We're using that to avoid any possible route typos 😄 Nice to see it over here!